utils/settings.py

```diff
@@ -107,13 +126,15 @@ def check_vars(path, checks):
     crawl_and_check(config, path, checks)
 
 
-def check_toml(template_file, config_file) -> Tuple[bool, Dict]:
+def check_toml(template_file, config_file) -> Dict[str, Any] | None | Literal[False]:
```

Comment:
What's the cause of this sig change?

Reply:
this function never returns bool and dict like

```
somebool, config = check_toml()
```

I have corrected them, next pr with change everything here btw